libs/npm/core/src/hooks/summon/useSummon.tsx

```diff
@@ -96,7 +96,8 @@ export const useSummon = ({
       },
     );
 
-    (axiosInstances as any)[baseURL ?? 'default'] = instance;
+    (axiosInstances as any)[baseURL ?? 'default'] = summon;
+    instance.current = summon;
```

Comment:
This is required so that the first attempt to use the summon hook doesn't return `undefined`